src/test/java/org/elasticsearch/test/rest/client/RestClient.java

```diff
@@ -50,6 +52,8 @@
 public class RestClient implements Closeable {
 
     private static final ESLogger logger = Loggers.getLogger(RestClient.class);
+    //query_string params that don't need to be declared in the spec, thay are supported by default
+    private static final Set<String> DEFAULT_QUERY_STRING_PARAMS = Sets.newHashSet("pretty", "source", "filter_path");
```

Comment:
sorry for nitpicking, just my deep dislike for naming things default/simple/fast :)... `ALWAYS_ACCEPTED_QUERY_STRING_PARAMS` or sth?